github-runner-provisioner/requesthandler.go

```diff
@@ -16,7 +16,7 @@ func handleProvisioningRequest(w http.ResponseWriter, r *http.Request) {
 	if !strings.HasPrefix(r.URL.String(), "/github-runner-provisioner") {
 		message := fmt.Sprintf("URL %s is invalid", r.URL.String())
 		http.Error(w, message, http.StatusBadRequest)
-		log.Printf(message)
+		log.Errorf(message)
```

Comment:
Context like HTTP request would be very helpful. I don't know how complicated will that be, so since this is for a CA, feel free to skip